### PR TITLE
Navigate rooms

### PR DESCRIPTION
**Pull request by MeenaFoerderer**
I finished my second user story so that the user is able to navigate between the 3 conference rooms and see the respective sessions and talks. I built 3 pages (one for each room), used filter and map methods to display the talks, added a `Navbar` that has previous and next buttons, and gave it and the header a fixed position.

### CODE REVIEW

File: pages/greatHall/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                <p>{talk.time}</p>
+              </TalkInfoWrapper>
+            </TalkItem>
+          ))}
+      </TalkList>
+      <NavBar>
+        <Link href="../libertyHall">
+          <NavButton>Prev</NavButton>
+        </Link>
+        <Link href="../metroEast">
```
Comment (gaiisay) [inline pages/greatHall/index.js:33]:
```suggestion
        <Link href="/libertyHall">
          <NavButton>Prev</NavButton>
        </Link>
        <Link href="/metroEast">
```

In den link tags musst du keinen relativen Pfad angeben. Der startet immer im root Ordner.

File: pages/libertyHall/index.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                <p>{talk.time}</p>
+              </TalkInfoWrapper>
+            </TalkItem>
+          ))}
+      </TalkList>
+      <NavBar>
+        <Link href="../metroEast">
+          <NavButton>Prev</NavButton>
+        </Link>
+        <Link href="../greatHall">
```
Comment (gaiisay) [inline pages/libertyHall/index.js:34]:
```suggestion
        <Link href="/metroEast">
          <NavButton>Prev</NavButton>
        </Link>
        <Link href="/greatHall">
```

siehe oben

File: pages/metroEast/index.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                <p>{talk.time}</p>
+              </TalkInfoWrapper>
+            </TalkItem>
+          ))}
+      </TalkList>
+      <NavBar>
+        <Link href="../greatHall">
+          <NavButton>Prev</NavButton>
+        </Link>
+        <Link href="../libertyHall">
```
Comment (gaiisay) [inline pages/metroEast/index.js:34]:
```suggestion
        <Link href="/greatHall">
          <NavButton>Prev</NavButton>
        </Link>
        <Link href="/libertyHall">
```

siehe oben

**Review by gaiisay — CHANGES_REQUESTED** [review]
Your user story says that 
- in each room the talks are sorted by session
- in each session the talks are sorted by time

But you did not write any code which would accomplish this.
You pre-sorted your talks in the dummy data file.

File: pages/greatHall/index.js (hunk 1 above)
Comment (gaiisay) [reply pages/greatHall/index.js:33]: Bzw im pages ordner

**Comment by MeenaFoerderer** [comment]
Good review, thanks Gai! I implemented the suggested changes.

**Review by gaiisay — APPROVED** [review]
Sieht gut aus :)

**Pull request merged**